docs/core/rid-catalog.md

```diff
@@ -1,136 +1,176 @@
+
+When NuGet restores packages, it tries to find an exact match for the specified runtime.
+If an exact match is not found, NuGet walks back the graph until it finds the closest compatible system according to the RID graph.
+
+The following example is the actual entry for the `osx.10.11-x64` RID:
 
 ```json
 "osx.10.11-x64": {
     "#import": [ "osx.10.11", "osx.10.10-x64" ]
 }
 ```
-The above RID specifies that `osx.10.11-x64` imports `osx.10.10-x64`. This means that when restoring packages, NuGet will
-be able to restore packages that specify that they need `osx.10.10-x64` on `osx.10.11-x64`.
 
-A slightly bigger example RID graph:  
+The above RID specifies that `osx.10.11-x64` imports `osx.10.10-x64`. So, when NuGet restores packages, it tries to find an exact match for  `osx.10.11-x64` in the package. If NuGet cannot find the specific runtime, it can restore packages that specify `osx.10.10-x64` runtimes, for example.
 
-- `win10-arm`
-  - `win10`
-  - `win81-arm`
-    - `win81`
-    - `win8-arm`
-      - `win8`
-        - `win7`
-          - `win`
-            - `any`
+The following example shows a slightly bigger RID graph also defined in the *runtime.json*  file:
+
+```
+    win7-x64    win7-x86
+       |   \   /    |
+       |   win7     |
+       |     |      |
+    win-x64  |  win-x86
+          \  |  /
+            win
+             |
+            any
+```
 
 All RIDs eventually map back to the root `any` RID.
 
-Although they look easy enough to use, there are some special things about RIDs that you have to keep in mind when 
-working with them:
+There are some considerations about RIDs that you have to keep in mind when working with them:
 
-* They are **opaque strings** and should be treated as black boxes
-    * You should not construct RIDs programmatically
-* You need to use the RIDs that are already defined for the platform and this document shows that
-* The RIDs do need to be specific so don't assume anything from the actual RID value; please consult this document 
-to determine which RID(s) you need for a given platform
+- RIDs are **opaque strings** and should be treated as black boxes.
+- Don't build RIDs programmatically.
+- Use RIDs that are already defined for the platform.
+- The RIDs need to be specific, so don't assume anything from the actual RID value.
 
 ## Using RIDs
-In order to use RIDs, you have to know which RIDs there are. New RIDs are added regularly to the platform. 
-For the latest version, please check the [runtime.json](https://github.com/dotnet/corefx/blob/master/pkg/Microsoft.NETCore.Platforms/runtime.json) file on CoreFX repo.
 
-> [!NOTE]
-> We are working towards getting this information into a more interactive form. 
-> When that happens, this page will be updated to point to that tool and/or its usage documentation. 
+To be able to use RIDs, you have to know which RIDs exist. New values are added regularly to the platform.
```

Comment:
There is a way for a package to extend the graph if needed. Should we doc that?

Reply:
@robrrelyea / @emgarten?  I recall that previous NuGet owners didn't want to doc this.  Now with the prevalence of distros we may need to.

Reply:
Dependencies through runtime.json is considered something that only system packages should use. I'm not sure on the RID graph itself which is also part of runtime.json.

Reply:
I'm a bit confused on what exactly to do here. cc @rrelyea